.gas-snapshot

```diff
@@ -111,14 +111,14 @@ L1AAVEStratTest:testTVL() (gas: 426307)
 L1BridgeEscrowTest:testclearFunds() (gas: 193033)
 L1BridgeEscrowTest:testclearFundsInvariants() (gas: 159376)
 L1BridgeEscrowTest:testclearFundsWithBadProof() (gas: 190575)
-L1DeltaNeutralTest:testClaimAndSellSushi() (gas: 1135395)
-L1DeltaNeutralTest:testCreatePosition() (gas: 1003827)
-L1DeltaNeutralTest:testDivest() (gas: 1318384)
-L1DeltaNeutralTest:testEndPosition() (gas: 1206346)
-L1DeltaNeutralTest:testOnlyAddressWithStrategistRoleCanEndPosition() (gas: 1038162)
+L1DeltaNeutralTest:testClaimAndSellSushi() (gas: 1129960)
```

Comment:
I don't think some of these gas costs should be changing. Can you run `foundryup`?